util/storage-samples/storage-owner-sample/src/main/java/org/fido/iot/storage/OwnerSysModule.java

```diff
@@ -142,10 +142,17 @@ private Composite getNextMessage(Composite resource) {
 
     message.set(Const.FIRST_KEY, contentType);
     switch (contentType) {
-      case FdoSys.KEY_ACTIVE:
-        message.set(Const.SECOND_KEY, true);
+      case FdoSys.KEY_ACTIVE: {
+
+        byte[] content = new OwnerDbManager().getSystemResourceContent(dataSource, resId);
+        if (content.length == 1 && content[0] == -11) {
```

Comment:
What is -11 here? Shouldn't it be F4 or F5 as a CBOR byte?

Reply:
Hex value F5 is -11 when read as signed character.

Reply:
We should add a comment. Also, could we use 0xF5 instead of -11.